cmd/syncthing/gui.go

```diff
@@ -1019,8 +1019,15 @@ func (s *apiService) getEvents(w http.ResponseWriter, r *http.Request, eventSub
 	qs := r.URL.Query()
 	sinceStr := qs.Get("since")
 	limitStr := qs.Get("limit")
+	timeoutStr := qs.Get("timeout")
 	since, _ := strconv.Atoi(sinceStr)
 	limit, _ := strconv.Atoi(limitStr)
+	var timeoutSec int
+	if timeoutStr == "" {
+		timeoutSec = 60 // Default to 60 for backwards compatibility
+	} else {
+		timeoutSec, _ = strconv.Atoi(timeoutStr)
+	}
```

Comment:
It might be better to always try and parse the time (even if its an empty string) and then always default to 60s in case of error.